pkg/converter/convert_unix.go

```diff
@@ -700,6 +700,16 @@ func IsNydusBlob(ctx context.Context, desc ocispec.Descriptor) bool {
 	return hasAnno
 }
 
+// IsNydusBootstrap returns true when the specified descriptor is nydus bootstrap layer.
+func IsNydusBootstrap(ctx context.Context, desc ocispec.Descriptor) bool {
```

Comment:
The `ctx` is not used, should we remove it?

Reply:
Fixed.